train.gemspec

```
@@ -31,7 +31,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "train-winrm", "~> 0.2"
 
   # azure, docker, gcp dependencies
-  spec.add_dependency "activesupport", ">= 5.2.4.3", "< 6.0.0" # ActiveSupport 6+ does not support ruby 2.4
+  spec.add_dependency "activesupport", ">= 6.0.0"
```

For security reasons, [this should be >=6.0.3.1](https://snyk.io/vuln/SNYK-RUBY-ACTIVESUPPORT-569598)

```suggestion
  spec.add_dependency "activesupport", ">= 6.0.3.1"
```